test/avocado/run-tests

```diff
@@ -301,6 +301,8 @@ def main():
         "selenium-machines-consoles.py",
         "selenium-machines-disks.py",
         "selenium-machines-networks.py",
+        "selenium-machines-nonroot.py",
```

Comment:
For the future please add each test in separate commit for readability purposes.

Reply:
OK, i will pay attention to it later, many thanks